pocs/camera/canon_gphoto2.py

```diff
@@ -1,3 +1,4 @@
+import datetime
```

Comment:
Not terribly important here, but we try to make all date/time items come from `current_time` (which allows us to override time in a simulator mode). Could be argued that here you don't ever want to do that and we always want the current.

Reply:
Done.